simd/unit_tests/include/SIMDTesting_Utilities.hpp

```diff
@@ -181,4 +181,10 @@ class load_as_scalars {
   }
 };
 
+template <typename T, typename = void>
+constexpr bool is_type_v = false;
+
+template <typename T>
+constexpr bool is_type_v<T, decltype(void(sizeof(T)))> = true;
```

Comment:
What is this used for? Would you mind adding some comments?

Reply:
This is to loosely check that the type `T` is a complete type.
Because of how `abi_set` and `data_type_set` are currently defined, not all data types can be paired with an abi type with an extended vector width. So, this check is simply used to skip compiling tests for those datatype+abi pairs that are not defined.

Reply:
Would you mind adding a comment to that effect?

Reply:
Added a comment explaining the use case of these structs.